lib/wings/active_fedora_converter.rb

```diff
@@ -141,11 +142,19 @@ def apply_attributes_to_model(af_object)
         members = Array.wrap(converted_attrs.delete(:members))
         files = converted_attrs.delete(:files)
         af_object.attributes = converted_attrs
+        if resource.try(:lease) && af_object.reflections.include?(:lease)
+          # NOTE: af_object.lease.class has the same name as resource.lease.class; however, each class has a different object_id
```

Comment:
Ah yes...antics.  I remember these.  There's definitely a subtle issue elsewhere that is beyond the scope of this work.